src/kibana/components/vislib/visualizations/area_chart.js

```diff
@@ -82,22 +82,18 @@ define(function (require) {
       })
       .interpolate(interpolate);
 
-      var layer;
-      var path;
-
       // Data layers
-      layer = svg.selectAll('.layer')
+      var layer = svg.selectAll('.layer')
       .data(layers)
-      .enter().append('g')
+      .enter()
+      .append('g')
       .attr('class', function (d, i) {
-        return i;
+        return 'pathgroup ' + i;
```

Comment:
I like that the class is now describing what type of element it is, but what is the `i` for here? Is this another instance of storing values in a class name?